tests/index-test.js

```diff
@@ -28,7 +28,14 @@ function matches(source, expected) {
 // Ensure each of the config mappings is mapped correctly
 Object.keys(mapping).forEach(global => {
   const imported = mapping[global];
-  let [importRoot, importName] = imported;
+  const importRoot = imported[0];
+
+  // Only process @ember imports
+  if (importRoot.indexOf('@ember/') === -1) {
```

Comment:
Non-blocking nitpick: I think we could use `importRoot.startsWith('@ember')` here for some yummy ES6 goodness.